### PR TITLE
Support setting inner html on an element

### LINKED ISSUE #117: Support setting innerHTML on Client
When a string of HTML is inserted into a view, only the server renders the HTML properly.  The client will display the string of HTML with escaped characters, showing the HTML tags as text.

This can be easily replicated by inserting `{ "<strong>test</strong>" }` into the output of some view's render function.  It would be nice to have the client render such strings similarly by setting `innerHTML`.

### PR DESCRIPTION
**Pull request by chinedufn**
Closes #117

### CODE REVIEW

File: book/src/html-macro/setting-inner-html/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Setting Inner HTML
```
Comment (chinedufn) [inline book/src/html-macro/setting-inner-html/README.md]: image

**Comment by austinsheep** [comment]
Awesome! I gave it a run and it works as expected 🎉 

Thanks for the fast implementation.

File: book/src/html-macro/setting-inner-html/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+You'll sometimes want to use a string of HTML in order to set the child nodes for an element.
+
+For example, if you're creating a tooltip component you might want to be able to support setting tooltips as such:
+
+<div data-tip="Hello <strong>World!</strong>"></div>
+
+You can use the `unsafe_inner_html` attribute for this purpose.
+
+Note that it is called `unsafe` because it can poentially expose your application to cross side scripting attacks if your application
```
Comment (austinsheep) [inline book/src/html-macro/setting-inner-html/README.md]: *cross-site

Comment (chinedufn) [reply book/src/html-macro/setting-inner-html/README.md]: Thanks - fixed 7fb1095

**Comment by chinedufn** [comment]
Awesome - thanks!

`virtual-dom-rs = "0.6.8"`

**Pull request merged**